docs/src/examples/components/Popup/Types/index.tsx

```diff
@@ -20,11 +20,6 @@ const Types = () => (
       description="Note that if Popup is controlled, then its 'open' prop value could be changed either by parent component, or by user actions (e.g. key press) - thus it is necessary to handle 'onOpenChange' event. Try to type some text into popup's input field and press ESC to see the effect."
       examplePath="components/Popup/Types/PopupControlledExample"
     />
-    <ComponentExample
-      title="Content Wrapper"
-      description="Use 'content' prop of the Popup to set whether Popup content should be rendered with the default wrapper."
-      examplePath="components/Popup/Types/PopupContentWrapperExample"
-    />
```

Comment:
This example again show the trick as working pattern, it's definitely not related to `Types` section.